packages/tabs/__tests__/tabs.test.tsx

```diff
@@ -51,6 +51,56 @@ describe("<Tabs />", () => {
     fireEvent.keyDown(tabList, { key: "Home", code: 36 });
     expect(document.activeElement).toBe(container.querySelector(".tab-1"));
   });
+
+  it("should match the snapshot (vertical orientation)", () => {
+    const { asFragment } = render(<VerticallyOrientedTabs />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("focuses the correct tab with keyboard navigation (vertical orientation)", () => {
+    const { getByText, getByRole, asFragment, container } = render(
+      <VerticallyOrientedTabs />
+    );
+    const firstTab = getByText("Tab One");
+    const tabList = getByRole("tablist");
+
+    function getTabPanelByButtonClass(buttonId: string) {
+      return container.querySelector(
+        `#${container
+          .querySelector(`.${buttonId}`)!
+          .getAttribute("aria-controls")}`
+      );
+    }
+
+    fireEvent.click(firstTab);
+
+    fireEvent.keyDown(tabList, { key: "ArrowDown", code: 40 });
+    expect(document.activeElement).toBe(container.querySelector(".tab-2"));
```

Comment:
Strangely, I'm getting different results from [what I'm getting from Travis](https://travis-ci.com/reach/reach-ui/builds/150023686) when I run `yarn test` locally. For me, the tests fail here on line 78, with this output:
https://gist.github.com/peterjmag/cfd150fb92a04635c5d1a58921edd197

Digging into this too.

Reply:
Turns out I had some outdated built files in dist/ when I last updated the snapshots (and the tests run against those built files). I re-ran the build and then updated the snapshots again in https://github.com/reach/reach-ui/pull/486/commits/b10530560716985c963321a79e54500d6d8e31e9, but now I'm getting differing IDs from Travis:

```
              <button
    -           aria-controls="tabs--15--panel--0"
    +           aria-controls="tabs--REACH_ID_--panel--0"
                aria-selected="true"
                class="tab-1"
                data-reach-tab=""
                data-selected=""
    -           id="tabs--15--tab--0"
    +           id="tabs--REACH_ID_--tab--0"
                role="tab"
                tabindex="0"
              >
```
https://travis-ci.com/reach/reach-ui/builds/150269550#L431

Reply:
Ended up just adding an ID to the test component itself to satisfy Travis: 
https://github.com/reach/reach-ui/pull/486/commits/953169dc466a0f8f934d0bfc6d45f19a90f839c1#diff-a65e1b1f6a8ab0f93d7a3e2afee4d44eR138